### ISSUE TITLE
Cloudinary sign-in window does not open in Firefox or Safari browsers

### ISSUE THREAD

**Issue by Nickm615** [kontent-ai/integration-cloudinary#8]
### Brief bug description
When attempting to sign in to Cloudinary using the custom element in firefox, the external window cannot open. The Firefox browser console displays the following message
![image](https://user-images.githubusercontent.com/86981780/206738743-60aa99a7-c23d-4fcb-928f-48e8730b9fe9.png)

### Repro steps
1. Open an item containing the Cloudinary custom element
2. Click on the sign in button

### Expected behavior

A new window opens and lets the user log in

### Test environment

Firefox v 107.0.1

**Comment by Nickm615** [comment]
The customer who reported this problem is also reporting similar issues in Safari browser, but I'm unable to test for the time being as there is no supported Windows version

**Comment by laststandjam** [comment]
Tried it on safari and was unable to load some resources due to "requested insecurely"

![Screen Shot 2022-12-09 at 8 15 07 AM](https://user-images.githubusercontent.com/53449817/206748801-478e9cc3-cd6a-409f-a971-edfd5697f9a5.png)

**Comment by JiriLojda** [comment]
Hi,
Thank you for submitting the issue. It seems there are two issues here.
First is the `iframe` that wraps the custom element (it is part of the Kontent.ai app) is sandboxed and doesn't allow `allow-storage-access-by-user-activation` permission. Which means the Cloudinary widget cannot access the browser storage which causes it to fail. After we consulted adding the permission with our security team, I was able to add it and it should be released next week. It should fix the integration for Firefox users.
Second issue is connected to Safari. The custom element is built using the official Cloudinary widget (https://cloudinary.com/documentation/media_library_widget). Unfortunately, it seems the widget doesn't work properly on Safari. I tried the example in their documentation and it doesn't seem to work either. We will try to contact them about the issue, but in the meantime the workaround is to open the custom element in a different browser. After the next week's release Firefox, Chrome and all the Chromium based browsers (Edge, Opera...) should work properly.
